### PR TITLE
Add POI nodes from wikidata to graph

### PR DESCRIPTION
**Pull request by jpalowitch**

### CODE REVIEW

File: cabby/geo/map_processing/map_structure.py [hunk 1]
```diff
@@ -122,6 +122,12 @@ def add_single_poi_to_graph(
     Returns:
       The new osmid.
     '''
+    # If the POI is already in the graph, do not add it.
+    if single_poi['osmid'] in self.nx_graph:
+      return None
+
+    print("processing new poi %d" % single_poi['osmid'])
```
Comment (tzufgoogle) [inline cabby/geo/map_processing/map_structure.py:129]: change print to logging.info

File: cabby/geo/map_processing/map_structure.py [hunk 2]
```diff
@@ -339,9 +344,9 @@ def add_two_ways_edges(self, edge_add: edge.Edge):
 
   def add_poi_to_graph(self):
     '''Add all POI to nx_graph(currently contains only the roads).'''
-    eges_to_add_list = self.poi.apply(self.add_single_poi_to_graph, axis=1)
-    
-    eges_to_add_list.swifter.apply(
+    edges_to_add_list = self.poi.apply(self.add_single_poi_to_graph, axis=1)
+    edges_to_add_list = edges_to_add_list.dropna()
```
Comment (tzufgoogle) [inline cabby/geo/map_processing/map_structure.py:348]: In what cases is this needed?

**Review by tzufgoogle — COMMENTED** [review]
Minor things.

File: cabby/geo/map_processing/map_structure.py (hunk 2 above)
Comment (jpalowitch) [reply cabby/geo/map_processing/map_structure.py:348]: In this PR, `add_single_poi_to_graph` now returns `None` if the POI is already in the graph. So this removes any entries in the series `edges_to_add_list` that are `None`.

**Review by tzufgoogle — APPROVED** [review]

File: cabby/geo/map_processing/map_structure.py (hunk 1 above)
Comment (jpalowitch) [reply cabby/geo/map_processing/map_structure.py]: Fixed by 1d7bdc2

**Pull request merged**